bin/build.sh

```diff
@@ -459,6 +462,16 @@ if [ ${BUILD_LOCKPWN} -eq 1 ] ; then
 fi
 
 
+if [ ${INSTALL_DEV_DEPENDENCIES} -eq 1 ] ; then
+  sudo apt-get install -y python3-pip
+  sudo pip3 install -U flake8
+  sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-${LLVM_SHORT_VERSION} 30
```

Comment:
Shouldn't clang format be installed first? It seems you are assuming it is already installed.

Reply:
Good catch. It seems clang-format is already installed by some step in our build script on the GitHub Actions machines.